Story Reader/main.py

```diff
@@ -0,0 +1,19 @@
+from time import sleep
+
+import pyttsx3 #pip install pyttsx3
+import PyPDF2 #pip install PyPDF2
+
+story = open('mybook.pdf', 'rb') #Name of your script make sure it's in pdf format
```

Comment:
Take the File Input from User. Don't hardcode it

Reply:
issue solved.  please review